### ISSUE TITLE
Thanks!

### ISSUE THREAD

**Issue by minhchau273** [thuctvd/TipCalculator#1]
Hi Thuc,
Thanks for your submission. It looks good. We are going to review and get back to you by next week.

**Comment by thuctvd** [comment]
Ok Thanks Ms Chau.

**Comment by minhchau273** [comment]
👍 Congratulation on finishing the Tip Calculator. Your app looks great with all of the optional features and the cool animation. 🎉 

#### Some small comments:
- Nice to see your clean and clear code. 💯 
- Good usage of the `NSUserDefaults`. This is one of the methods to pass data between view controllers.
- Good job to use `NSUserDefaults` to save remember the bill amount across app restarts. It'd be better if you saved the selected theme as well.
- You shouldn't let the user enter invalid number, should you? I can enter bunch of leading zeros and many dots. I can also paste some text in the text field:  
  <img width="304" alt="screen shot 2016-10-01 at 8 16 50 pm" src="https://cloud.githubusercontent.com/assets/12395213/19014423/a1c9c39a-8816-11e6-9057-f0d0e5f3e8a4.png">
- Great job with the locale-specific currency.
- Try to challenge yourself on the visual side to make your app more impressive and increase your chance. 😉

Excellent submission overall. 🌟  We will get back to you soon.

**Comment by thuctvd** [comment]
yes, i haven't validate bill input yet. I'll fix it .that is a bug, big bug ^^!
